### ISSUE TITLE
notification.CancelOperation message not showing

### ISSUE THREAD

**Issue by ksolberg** [umbraco/Umbraco-CMS#16794]
### Which Umbraco version are you using? (Please write the *exact* version, example: 10.1.0)

14.1.1

### Bug summary

Cancelling a ContentSaving event with a message, but only a generic error shows up.

![image](https://github.com/user-attachments/assets/01adfcb0-35bd-4895-a883-9a27d11e3e87)

![image](https://github.com/user-attachments/assets/6cc2c696-0b1e-43a7-9bee-3104e579d56c)

### Steps to reproduce

Implements a class that inherits INotificationHandler<ContentSavingNotification>. Loop through SavedEntities and call notification.CancelOperation(new EventMessager("title", "message", EventTypeMessage.Error). Operatinon is cancelled, but custom error title and message is not shown.

---
_This item has been added to our backlog AB#42404_

**Comment by elit0451** [comment]
Hi @ksolberg 👋 

Thank you for reaching out! I was able to reproduce this, we will have a look. Seems like we are not returning the notification message as we had in v13:

![Notification](https://github.com/user-attachments/assets/3adedb63-3380-4f02-bec6-241d1a539536)

Here is the example code that I used:

```c#
public class MyNotificationHandler : INotificationHandler<ContentSavingNotification>
{
    public void Handle(ContentSavingNotification notification)
    {
        foreach (IContent entity in notification.SavedEntities)
        {
            notification.CancelOperation(new EventMessage("My category", "We are canceling", EventMessageType.Error));
        }
    }
}

public class MyComposer : IComposer
{
    public void Compose(IUmbracoBuilder builder)
    {
        builder.AddNotificationHandler<ContentSavingNotification, MyNotificationHandler>();
    }
}
```

**Comment by elit0451** [comment]
Hello again, actually the notification message is part of the `Umb-Notifications` header and it is a known issue from the front-end side, so it is coming 🙂 

![CanceledMsg](https://github.com/user-attachments/assets/2ab3e8d7-97b2-4955-95b5-af8db4a543ee)

**Comment by DORE-PD** [comment]
Is there an update on this? I also have the problem in v13.4.1.
But I get a successful message, even if the content is not moved.

```
		public bool CanHandle(ContentMovingNotification notification)
		{
			return true;
		}

		public void Handle(ContentMovingNotification notification)
		{
			notification.CancelOperation(new EventMessage("System", "Not allowed", EventMessageType.Error));
		}
```

![image](https://github.com/user-attachments/assets/0425641a-e44a-457f-bc6c-d78c5362af5a)

**Comment by NguyenThuyLan** [comment]
I was able to reproduce it in v13.5.0 when using ContentMovingNotification , but it works with ContentSavingNotification. And I was not able to reproduce in V14.2.0

**Comment by iOvergaard** [comment]
This should be fixed in 14.2, but unfortunately I cannot link it back to any pull request. Feel free to re-open if the issue still occurs :-)
